agent_api_rest/src/identity/mod.rs

```diff
@@ -1,13 +1,16 @@
 pub mod connections;
+pub mod documents;
 pub mod services;
 pub mod well_known;
 
+use crate::identity::documents::get_documents;
 use agent_identity::state::IdentityState;
 use axum::{
     routing::{get, post},
     Router,
 };
 use connections::{get_connection, get_connections, post_connections};
+use documents::get_document;
```

Comment:
Since `get_documents` and `get_document` are located in the same file it would make sense here to combine both imports into one:
```rust
use documents::{get_document, get_documents};
```